parastell/parastell.py

```diff
@@ -457,6 +412,63 @@ def export_dagmc(
                 **kwargs
             )
 
+    def build_full_model(self, invessel_build, magnet_coils, source_mesh):
+        """Use all of the input to make a complete stellarator model
+        with invessel components, magnets, and a source mesh
+        """
```

Comment:
Should probably include some version of argument explanations in this docstring.